src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -308,6 +312,18 @@ protected ImagePullPolicy deduceImagePullPolicy(AppDeploymentRequest request) {
 		return requests;
 	}
 
+	/**
+	 * Get the hostNetwork setting for the deployment request.
+	 *
+	 * @param request The deployment request.
+	 * @return Whether host networking is requested
+	 */
+	protected boolean getHostNetwork(AppDeploymentRequest request) {
+		String hostNetwork =
+				request.getDeploymentProperties().get("spring.cloud.deployer.kubernetes.hostNetwork");
```

Comment:
Not on the scope of this PR: but, we can try to keep a constant for `spring.cloud.deployer.kubernetes` at some point for all the deployer properties prefix.